ducktape/cluster/remoteaccount.py

```diff
@@ -272,6 +272,12 @@ def _can_ping_url(self, url, headers):
         except Exception:
             return False
 
+    def available(self):
+        try:
+            return self.ssh("echo ping") == 0
```

Comment:
I checked `echo` docs on microsoft website and it should pretty much work the same way as it does on linux machines - printing its arguments and having exit code of 0. I can try to validate on my personal windows box too.

Good idea about just creating ssh client, I haven't tried whether or not it is enough, will check. I would probably still want to keep the `available` method though, just because of how easy it is to mock it for testing, can probably just do the implementation of creating ssh client in try and cleaning up in finally.

Reply:
Updated to just create ssh client. Also moved try-finally block to the available() func, to make it cleaner